test/google_crawler_web/controllers/api/keyword_controller_test.exs

```diff
@@ -282,5 +283,53 @@ defmodule GoogleCrawlerWeb.Api.KeywordControllerTest do
 
       assert length(keywords) == 3
     end
+
+    test "returns error when imports invalid file type", %{conn: conn} do
+      user = insert(:user)
+
+      reject(GoogleCrawler.Keywords.ScraperSupervisor, :start_child, 1)
+
+      uploaded_file = %Plug.Upload{content_type: "application/pdf"}
+
+      conn =
+        conn
+        |> login_as(user)
+        |> post(Routes.keyword_import_path(conn, :import), %{keywords: uploaded_file})
+
+      assert %{
+               "code" => "bad_request",
+               "object" => "error"
+             } = json_response(conn, 400)
```

Comment:
Minor, but since we're not pattern matching here, perhaps we could do
```elixir
json_response(conn, 400) == %{
               "code" => "bad_request",
               "object" => "error"
             }
```